### PR TITLE
fix(tests): fixed mac port issue in smoke tests

### LINKED ISSUE #5279: unable to execute smoke tests on macOS
### Bug description

it is failing on 5000 port number being already used when testing pods

### Operating system

macOS

### Installation Method

None

### Version

1.6.2

### Steps to reproduce

try to start a process listening on port 5000 first (on macOS it is the case)

then start the smoke tests

you should see the '5000 port already in use' issue

### Relevant log output

```shell
the process 'Control Center' of Apple is listening on that port so tests are failing
```

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by rostalan**
### What does this PR do?

This PR fixes mac issue, where the pod is not created, when port 5000 is already in use.
The port is instead set to 5101, which should not be used by default on mac.
I also fixed the pod test after button name change.

### Screenshot / video of UI

image

### What issues does this PR fix or reference?

This fixes [5279](https://github.com/containers/podman-desktop/issues/5279)

### How to test this PR?

`yarn test:e2e:smoke`

### CODE REVIEW

**Review by jeffmaury — APPROVED** [review]
LGTM

File: tests/src/model/pages/run-image-page.ts [hunk 1]
```diff
@@ -109,4 +109,11 @@ export class RunImagePage extends BasePage {
     await hostPort.fill(customPortMapping.split(':')[0]);
     await containerPort.fill(customPortMapping.split(':')[1]);
   }
+
+  async setHostPortMapping(customPort: string): Promise<void> {
```
Comment (odockal) [inline tests/src/model/pages/run-image-page.ts]: not necessary anymore: https://github.com/containers/podman-desktop/pull/5304/files#diff-c66b8072e74147f14715eb1f73a95799bc9deb70efd9ead3f3e32d32e4ed4f85R51

Comment (rostalan) [reply tests/src/model/pages/run-image-page.ts:113]: removed duplicit method

Comment (rostalan) [reply tests/src/model/pages/run-image-page.ts]: removed duplicit method

**Review by odockal — APPROVED** [review]

**Pull request merged**